packages/lesswrong/components/users/UsersNamePending.tsx

```diff
@@ -0,0 +1,53 @@
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import React from 'react';
+import { Link } from '../../lib/reactRouterWrapper';
+import ErrorIcon from '@material-ui/icons/ErrorOutline';
+import {
+  userGetDisplayName,
+  userGetProfileUrl,
+} from '../../lib/collections/users/helpers';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  userName: {
+    whiteSpace: "nowrap",
+    color: theme.palette.text.secondary,
+  },
+  tooltipUserName: {
+    fontWeight: "bold",
+  },
+  icon: {
+    transform: "translateY(4px)",
+  },
+});
+
+const UsersNamePending = ({ user, classes }: {
+  user: UsersMinimumInfo,
+  classes: ClassesType,
+}) => {
+  const { LWTooltip } = Components;
+  const name = userGetDisplayName(user)
+  const tooltip = <p>
+    You have requested <span className={classes.tooltipUserName}>{name}</span> to
+    co-author this post. They can accept or decline this request.
+  </p>;
+
+  return (
+    <span>
```

Comment:
Fair point - I think I just kept these because the original markup that I moved had a `span`, but you're absolutely right that it's not needed. I'll make the change 👍